examples/tools/use_python_node.py

```diff
@@ -0,0 +1,137 @@
+from pydantic import ConfigDict
```

Comment:
Please extend python node tests with the similar examples: https://github.com/dynamiq-ai/dynamiq/blob/main/tests/integration/nodes/tools/test_python.py